packages/eas-cli/src/commands/build/list.ts

```diff
@@ -51,6 +52,7 @@ export default class BuildList extends EasCommand {
     ...EasPaginatedQueryFlags,
     limit: getLimitFlagWithCustomValues({ defaultTo: 10, limit: BUILDS_LIMIT }),
     ...EasNonInteractiveAndJsonFlags,
+    simulator: Flags.boolean(),
```

Comment:
Maybe it would be good to add a description to this flag, putting emphasis on the fact that it will only filter the **iOS** simulator builds. People tend to confuse Android emulator with iOS simulator and one could think that this will also query for Android emulator builds.